cpp/include/gar/util/data_type.h

```diff
@@ -51,6 +51,9 @@ enum class Type {
   /** UTF8 variable-length string */
   STRING,
 
+  /** List of some logical data type */
```

Comment:
Could you please add LIST into the [data types](https://alibaba.github.io/GraphAr/file-format.html#data-types) and explain that CSV does not support it?